zadaci/sa-kolokvijuma/2020/PSI/T5/G4/zadatak.rst

```diff
@@ -0,0 +1,41 @@
+Proizvodi su opisani na sledeći način:
+
+* Naziv (jedna reč, maksimalno 20 karaktera)
+* vrsta (jedna reč, maksimalno 15 karaktera)
+* cena (realan broj)
+  
+Za proizvode iz ulazne datoteke uraditi sledeće:
+
+a) Učitati proizvode u jednostruko spregnutu listu. Novi elementi dodaju se tako da lista bude sortirana po ceni u rastućem redosledu.
+b) Na osnovu argumenta komandne linije koji označava vrstu proizvoda, na standardni izlaz (ekran Terminala) ispisati sledeće informacije:
+
+  * cena  (zaokruženo na dve decimale, ispisano sa ``%6.2lf`` format specifikatorom)
```

Comment:
Minor rendering issue: please add one more space in front of each `*` item marker to make them a nested list, as they're a block-quote right now. 

It's maybe more visible via GitHub RST render https://github.com/petarmaric/pjisp-zbirka-zadataka/blob/45273e59e7e124ab1bad3c9bde0cc457b11cbb35/zadaci/sa-kolokvijuma/2020/PSI/T5/G4/zadatak.rst